python/paddle/fluid/incubate/fleet/parameter_server/pslib/__init__.py

```diff
@@ -263,6 +263,19 @@ def shrink_dense_table(self, decay, scope=None, table_id=None):
                                                    decay)
         self._role_maker._barrier_worker()
 
+    def clear_model(self):
```

Comment:
dose this interface only clear sparse but not dense ? if so , "clear_sparse_model" may be more clear